src/components/screens/Tour.js

```diff
@@ -97,7 +98,7 @@ export const Tour = ({ data, navigation, route }) => {
           </View>
         )}
 
-        {!!augmentedReality && <AugmentedReality navigation={navigation} />}
+        {/* {!!augmentedReality && <AugmentedReality navigation={navigation} />} */}
```

Comment:
this was for local testing? i think it should be down here uncommented?

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/579/commits/799cbe77e5eca37c9114f903dd0c1d5d388c0e1a